src/script.js

```diff
@@ -1 +1,119 @@
+// Import the stylesheet
 import './style.css';
+
+// Select relevant HTML elements
+const form = document.getElementById('form'); // Get the form element
+const input = document.getElementById('new-item'); // Get the input element
+const todoList = document.getElementById('todo-list'); // Get the todo list element
+// const archiveBtn = document.getElementById('archive'); // Get the archive button element
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.